docs/developer-docs/integrations/sns/launching/launch-summary.md

```diff
@@ -0,0 +1,332 @@
+| Newly-created SNS canisters on the SNS subnet     | Pre-decentralization swap mode        | Initial developer neurons |  
+| Initial developer neurons                         | Pre-decentralization swap mode        | Dapp developer principal  |
+| Treasury account on the SNS Ledger                | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+
+
+- #### Step 7: The initial SNS developer neurons accepts or rejects the SNS proposal
+
+  The initial SNS developer neurons are declared in the initial parameters and available at SNS installation.
+
+  This is the **second of three** proposals that need to successfully pass for the process to continue.
+
+  Since this proposal is passed by the SNS DAO. When one owns SNS neurons or votes on SNS proposals, one is are part of the SNS DAO. It is common for people to vote on both NNS and SNS proposals from within the [NNS Frontend dapp](nns.ic0.app), but they are two separate DAOs.
+
+  :::info
+  If this SNS proposal passes, it does **not** guarantee the rest of the steps will complete.
+  :::
+
+
+If successful, at the end of stage, we have:
+
+|  Decentralization swap state                      | PENDING                               |  
+|-------------------                                | ----------------                      |
+
+
+|  Objects in an app subnet                                      | State                    |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| A dapp                                            | Operational                           | ~~Dapp developer principal~~ Newly-created SNS canisters | 
+
+|  Objects in the SNS subnet                                       | State                  |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| ~~A principal that can deploy to SNS subnet~~     | Revoked because it is 1-time use      | NA                        |
+| Newly-created SNS canisters on the SNS subnet     | Pre-decentralization swap mode        | Initial developer neurons |  
+| Initial developer neurons                         | Pre-decentralization swap mode        | Dapp developer principal  |
+| Treasury account on the SNS Ledger                | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+
+
+
+- #### Step 8: Anyone in the community submits an NNS proposal to start the decentralization swap.{#SNS-launch-step-startSwap}
```

Comment:
If we say "anyone" here, should we also do so in the first proposal submission (the same applies there)